pkgs/development/python-modules/flask-sockets/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, flask
+, gevent
+, gevent-websocket
+}:
+
+buildPythonPackage rec {
+  pname = "Flask-Sockets";
+  version = "0.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "072927da8edca0e81e024f5787e643c87d80b351b714de95d723becb30e0643b";
+  };
+
+  propagatedBuildInputs = [
+    flask
+    gevent
+    gevent-websocket
+  ];
+
```

please add comment as to why there's no tests, and use pythonImportsCheck to make sure at least importing isn't broken
```suggestion

  # upstream doesn't have any tests, single file
  doCheck = false;
  
  pythonImportsCheck = [
    "flask_sockets"
  ];
```